test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -12,6 +12,9 @@ describe('CSVSyncer', () => {
 		expect(CSVSyncer.aggregateStats).to.not.equal(undefined);
 		expect(new CSVSyncer().steps).to.not.equal(undefined);
 		expect(new CSVSyncer().sync).to.not.equal(undefined);
+
+		expect(CSVSyncer.dateValidator).to.not.equal(undefined);
+		expect(CSVSyncer.stringToDateConverter).to.not.equal(undefined);
```

Comment:
This does not belong into this test, because it doesn't belong to the Syncer interface